### PR TITLE
[HOT-39] Add colour related attributes for categories

### PR DESCRIPTION
**Pull request by Keerush**
JIRA: https://hotpotbuddy.atlassian.net/browse/HOT-39

Added color palettes for each category. As a default is the miscellaneous colors if no category matches or is missing.

### CODE REVIEW

File: client/src/utils/colors.tsx [hunk 1]
```diff
@@ -1,4 +1,67 @@
...
+      'EAA8A8',
+      'BB0000',
+      'CD0000',
+      'DE1616',
+      'EF2B2B',
+      'EC4040',
+      'E95555',
+      'E56B6B',
+      'E28080',
+      'AA000'
```
Comment (AndrewLin95) [inline client/src/utils/colors.tsx:15]: This one looks kinda off `'AA000'`

File: client/src/utils/colors.tsx [hunk 2]
```diff
@@ -1,4 +1,67 @@
-// TODO: map to color schemes
+// Color array from lightest to darkest shades
+const CATEGORY_COLORS = new Map([
```
Comment (AndrewLin95) [inline client/src/utils/colors.tsx:2]: Oh missing the "#" from these.

**Review by AndrewLin95 — COMMENTED** [review]
Looks pretty good!!!

**Review by AndrewLin95 — CHANGES_REQUESTED** [review]
Updated to request changes. Almost missed the "#"'s you mentioned in chat myself.

File: client/src/utils/colors.tsx (hunk 2 above)
Comment (Keerush) [reply client/src/utils/colors.tsx:2]: Yep, done.

File: client/src/utils/colors.tsx (hunk 1 above)
Comment (Keerush) [reply client/src/utils/colors.tsx:15]: Nice catch, missed a 0

**Review by AndrewLin95 — APPROVED** [review]
:shipit:

**Pull request merged**